### PR TITLE
Feature Home Page

### PR DESCRIPTION
**Pull request by abhijitdotsharma**
Home Page Created with -
- Navbar
- Hero Image
- Categories
- New Arrival Card

Temp-NouStore-HomePage

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (ozairkhan2810) [inline .vscode/settings.json:2]: can remove this file

File: css/navbar.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
+.navbar {
+    background-color: var(--color-black);
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-items: center;
+    padding: 1rem 1.5rem;
+    border-bottom: 0.3px solid white;
```
Comment (ozairkhan2810) [inline css/navbar.css:9]: use color variable for white

File: css/navbar.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    gap: 1rem;
+}
+
+.navbar a{
+  color: var(--color-white);
+  font-size: 1.5rem;
+}
+.navbar i{
+  color: var(--color-white);
+}
```
Comment (ozairkhan2810) [inline css/navbar.css]: can combine similar properties to remove redundancy

File: css/navbar.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+.nav-btn{
+  padding: 0.5rem 1.5rem;
+  background-color: #9760d6;
```
Comment (ozairkhan2810) [inline css/navbar.css]: use color variable

**Review by ozairkhan2810 — APPROVED** [review]
suggested few changes, then merge

**Review by sks-21 — APPROVED** [review]
Good to merge

File: css/navbar.css (hunk 2 above)
Comment (abhijitdotsharma) [reply css/navbar.css:9]: I have a color white variable(#EEEEEE)  for reuse, here I wanted white only(#FFFFFF) once so making a new variable didn't seem ideal. Thank you for the suggestion anyway.

File: css/navbar.css (hunk 3 above)
Comment (abhijitdotsharma) [reply css/navbar.css]: Did that, thanks for pointing it out

File: css/navbar.css (hunk 4 above)
Comment (abhijitdotsharma) [reply css/navbar.css]: Did this, thanks

**Pull request merged**